bin/docker-disk-usage

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env php
+<?php
+
+use BiffBangPow\DockerDiskUsage\DockerDiskUsage;
+
```

Comment:
This is good @catharsisjelly, took no time at all to get the volumes with that, thanks for the heads up. It seems we still need to execute a bash style command to get the six of the volumes, but getting the rest of the info is much cleaner. I'll crack on with the containers and images.